src/gtk/webview_webkit2.cpp

```diff
@@ -1092,14 +1097,107 @@ wxString wxWebViewWebKit::GetPageText() const
     }
     return wxString();
 }
+static void wxgtk_run_javascript_cb(WebKitWebView *,
+                                    GAsyncResult *res,
+                                    GAsyncResult **res_out)
+{
+    *res_out = (GAsyncResult*)g_object_ref(res);
```

Comment:
Should use `static_cast<>` instead of C-style cast. Better yet, avoid the cast completely, why can't we do

    g_object_ref(res);
    *res_out = res;

instead?